### DISCUSSION TITLE
rtorrent/rutorrent can only handle 20 .torrent files at once (for re-check?) #1148

### DISCUSSION THREAD

**Discussion by blixten85** [Novik/ruTorrent#2644] [Ideas]
I would like this number to be increased, if a re-install has been done and the torrents are about to get re-added this program can only handle 20 of them at a time.
It's annoyoing to split it up.

I re-post it here because it might be more appropriate

**Comment by n1ce-0ne** [comment]
your lucky mine only does 1 at a time lololol
